Cargo.toml

```diff
@@ -13,5 +13,13 @@ log = "0.4"
 async-trait = "0.1"
 eyre = "0.6"
 color-eyre = "0.6"
+aws-config = "0.51"
+aws-sdk-s3 = "0.21"
+bytes = "*"
```

Comment:
This should not be `*`

Reply:
its using the version provided in aws-sdk-s3 though?

Reply:
Can you just stick a little doc on that